envs/6.0.0.env

```diff
@@ -1,4 +1,4 @@
-CHAIN_IMAGE=polymeshassociation/polymesh:99dc6cc1b2-develop-debian
-SUBQUERY_INDEXER_IMAGE=polymeshassociation/polymesh-subquery:v9.9.0
+CHAIN_IMAGE=polymeshassociation/polymesh:6.0.0-develop-debian
```

Comment:
This is a 3 month old image and isn't the final 6.0 release. 

Newer develop branch only use the commit hash. This should be `testnet` except for when testing the "next" release.